services/movie.py

```diff
@@ -1,19 +1,22 @@
+from django.db import transaction
 from django.db.models import QuerySet
 
 from db.models import Movie
 
 
 def get_movies(
+    title: str = None,
```

Comment:
It accepts Optional[str] because default parameter is None
Check in all places(only your code)